src/FeatureFlags/FeatureFlagsServiceProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\FeatureFlags;
+
+use Give\FeatureFlags\OptionBasedFormEditor\OptionBasedFormEditor;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\Advanced;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\DefaultOptions;
+use Give\FeatureFlags\OptionBasedFormEditor\Settings\General;
+use Give\Helpers\Hooks;
+use Give\ServiceProviders\ServiceProvider;
+
+/**
+ * @unreleased
+ */
+class FeatureFlagsServiceProvider implements ServiceProvider
+{
+    /**
+     * @unreleased
+     */
+    public function register()
+    {
+    }
+
+    /**
+     * @unreleased
+     */
+    public function boot()
+    {
+        $this->maybeDisableOptionBasedFormEditorSettings();
```

Comment:
As the `FeatureFlag` domain grows, we might want to consider the use of sub-domains to organize the specific feature flag implimentations, ie `src/FeatureFlags/OptionBasedEditor/ServiceProvider.php`.

Reply:
Resolved in cbb1de1570aced5893f840246d1db0bfebc18fc0